pyne/material.pyx

```diff
@@ -345,6 +345,19 @@ cdef class _Material:
         cdef std_string card
         card = self.mat_pointer.mcnp(frac_type.encode())
         return card.decode()
+    
+    def gdml(self):
```

Comment:
Docstring of this function is not clear. 
- Parameters are usually before Return.
- There isn't an input paremeter, what's the `int 0`?

Reply:
sorry, I forgot the clean the copy paste :)

thx for the catch!